shared/wallets/trustline/asset.tsx

```diff
@@ -0,0 +1,153 @@
+import * as React from 'react'
+import {memoize} from '../../util/memoize'
+import * as Kb from '../../common-adapters'
+import * as Styles from '../../styles'
+
+export type Props = {
+  code: string
+  expanded: boolean
+  firstItem: boolean
+  issuerAccountID: string
+  issuerVerifiedDomain: string
+  trusted: boolean // TODO add limit when we support it in GUI
+
+  onAccept: () => void
+  onCollapse: () => void
+  onExpand: () => void
+  onRemove: () => void
+  onViewDetails: () => void
+}
+
+const stopPropagation = memoize(onClick => e => {
```

Comment:
Why memoize this

Reply:
I was thinking to avoid making new functions if onClick from dispatchProps doesn’t change. Though now I’m realizing this doesn’t work since all the onClicks are using the same memoized function. Guess I’ll just get rid of it